src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -52,7 +55,7 @@ public boolean equals(Object other) {
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .add("predicate", predicate)
-                .toString();
+            .add("predicate", predicate)
+            .toString();
```

Comment:
Should the indentation here be 8 spaces instead? As the indentation for wrapped lines is usually 8 spaces more than the parent line according to the Java coding standard.